src/ApiCheck/BaselineComparer.cs

```diff
@@ -0,0 +1,138 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using ApiCheck.Baseline;
+using Microsoft.Extensions.CommandLineUtils;
+
+namespace ApiCheck
+{
+    public class BaselineComparer
+    {
+        private readonly BaselineDocument _newBaseline;
```

Comment:
Is ApiDescription good enough or your prefer ApiListing (or something else) I don't have a strong opinion about any of them.